paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/cvcrecollection/CvcRecollectionScreen.kt

```diff
@@ -86,6 +85,54 @@ internal fun CvcRecollectionScreen(
     }
 }
 
+@Composable
+internal fun CvcRecollectionPaymentSheetScreen(
+    cardBrand: CardBrand,
+    lastFour: String,
+    viewActionHandler: (action: CvcRecollectionViewAction) -> Unit
+) {
+    val element = rememberElement(cardBrand) { cvcState ->
+        viewActionHandler(CvcRecollectionViewAction.CvcStateChanged(cvcState))
+    }
```

Comment:
This logic should also all live in the interactor. Since the interactor has a state, we should be getting the state from the interactor and also updating the interactor's state based on changes in the UI.

I'd expect this to look more like this:
- the composable function takes in a `CvcRecollectionInteractor` as a parameter
- we get the current state (cardBrand, etc) by doing something like `val state by interactor.state.collectAsState()`
- and then when the text field in the UI is updated, we call `interactor.handleViewAction(cvcState)`

[ManageScreenUI](https://github.com/stripe/stripe-android/blob/ad6c0bba4220d7311d48d492a31755c63aec92cd/paymentsheet/src/main/java/com/stripe/android/paymentsheet/verticalmode/ManageScreenUI.kt#L4) is a good concise example of what this pattern looks like

Reply:
Once you do this, it should also be possible to write some UI tests, which can use the fake interactor to verify that when the CVC input field changes, then we call the right view action.

Let me know if it ends up being tricky to do these changes, since I noticed that the CVC UI uses a controller `ComposeUI` which might be harder to work with